Source/Scene/Model.js

```diff
@@ -119,7 +119,7 @@ define([
 
         this.decompressedViewsToCreate = new Queue();
         this.decompressedViews = {};
-        this.decompressionInFlight = 0;
+        this.decompressionInFlight = false;
```

Comment:
That was already the case (as is the case in imagery).  The only case where we use span multiple workers right now is the code in Primitive.js.  We have a long term item for a general thread pool to avoid contention.  I would saying trying to use multiple-workers for decompression now might not really improve anything (since we are already seeing contention with imagery.)

That all being said, turning this back into a counter and using multiple workers is really easy, so if you want me to do it (for performance experiments), it's probably only about 5 minutes worth of work.

There's also further improvements we can make when we start doing things like aborting load because a model that was loading is already no longer in view.  My emphasis here was just making sure everything worked correctly for now without spending lots of time on it.